integration/tests/basic/rpc/test_neon_rpc_base_calls.py

```diff
@@ -110,3 +110,24 @@ def test_neon_get_solana_transaction_by_neon_transaction_non_existent_tx(self, j
         )
         assert "error" not in response
         assert len(response["result"]) == 0, "expected empty result for non existent transaction request"
+
+    def test_neon_get_native_token_list(self, pytestconfig, json_rpc_client):
+        response = json_rpc_client.send_rpc(method="neon_getNativeTokenList")
+        assert "error" not in response
+        assert len(response["result"]) > 1, f"Expected non-empty list, got {response['result']}"
```

Comment:
you can check that it is the same count as in pytestconfig.environment.network_ids

Reply:
Fixed ✅